pkg/init/cmd/service/prepare.go

```diff
@@ -14,15 +14,11 @@ import (
 	"golang.org/x/sys/unix"
 )
 
-const (
-	wgPath      = "/usr/bin/wg"
-	nsenterPath = "/usr/bin/nsenter-net"
-)
-
```

Comment:
It built in CI so I suppose this is intentional/harmless?

Reply:
yeah I just happened to spot that, its not used any more...